python/vowpalwabbit/pyvw.py

```diff
@@ -1134,86 +1134,83 @@ def __str__(self):
         return " ".join(["{}:{}".format(c.label, c.cost) for c in self.costs])
 
 
-class cbandits_label(abstract_label):
+class cb_class:
+    def __init__(
+        self, action=None, cost=0.0, partial_prediction=0.0, probability=0.0, **kwargs
+    ):
+        if kwargs.get("label", False):
+            action = kwargs["label"]
+            warnings.warn(
+                "label has been deprecated. Please use 'action' instead.",
+                DeprecationWarning,
+            )
+        self.label = action
+        self.action = action
+        self.cost = cost
+        self.partial_prediction = partial_prediction
+        self.probability = probability
+
+
+class CBandits_label(abstract_label):
```

Comment:
This is a potential breaking change so we should change the name but add a deprecated way to use the old name

Reply:
PEP8 states classes should use PascalCase

Reply:
I'll add this in a separate PR with some other classes

Reply:
Okay, best to avoid changing this name in this PR then